d3a_interface/constants_limits.py

```diff
@@ -77,21 +78,22 @@ class StorageSettings:
         # Default value 1 stands for percentage/RISK based energy rate decrease
         # Option 2, use the constant energy rate decrease
         RATE_DECREASE_OPTION = 1
-        # Energy sell break-even point, storage never sells for less than this value.
+        # Energy buy-range, storage never buys outside this limit.
         # Unit is ct/kWh.
-        BREAK_EVEN_SELL = 25
-        # Energy buy break-even point, storage never buys for more than this value.
+        BUYING_RANGE = [0, 24.9]
+        INITIAL_BUYING_RANGE = [0, 1000]
+        FINAL_BUYING_RANGE = [0, 1000]
+        # Energy sell-range, storage never sell outside this limit.
         # Unit is ct/kWh.
-        BREAK_EVEN_BUY = 24.9
-        # Minimum acceptable buy rate for the battery, in ct/kWh.
-        MIN_BUYING_RATE = 24.9
-        # Maximum acceptable sell rate for the battery, in ct/kWh.
-        MAX_SELLING_RATE = 30
-        # Min allowed battery SOC, range is [0, 0.1].
-        MIN_ALLOWED_SOC = 0.1
+        SELLING_RANGE = [30, 25]
```

Comment:
Please add some descriptive comments here ( for all new settings, apart from the ranges). Thanks!

Reply:
EDIT, forget it. Its there. The diff view confuses me sometimes. Thanks!